.eslintrc.js

```diff
@@ -3,8 +3,10 @@
 // or overridden for a particular subset of the project.  See
 // other .eslintrc.js files for those rules.
 module.exports = {
+  "parser": "babel-eslint",
```

Comment:
as I understand it, this makes eslint use babel to parse the javascript files. babel in turn uses the a plugin to be able to parse flow type annotations. So this change makes eslint stop complaining about flowtype syntax which it doesn't understand by default.